src/aks-preview/HISTORY.rst

```diff
@@ -14,6 +14,10 @@ Pending
 * Vendor new SDK and bump API version to 2023-10-02-preview.
 * Update the enum for `--os-sku` in command `az aks nodepool update` to only accept the expected Ubuntu and AzureLinux OSSKUs.
 
+0.5.172
```

Comment:
please update `VERSION` in setup.py at the same time if you want to release a new version of aks-preview AND move all those notes under the pending section (several lines above) to this block.